modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -436,8 +436,7 @@ else if (StringUtil.startsWith(
 					null);
 			}
 			else {
-				results = objectValidationRuleEngine.execute(
-					(Map<String, Object>)variables.get("entryDTO"), null);
+				results = objectValidationRuleEngine.execute(variables, null);
```

Comment:
@devpedrotavares will this affect other implementations besides the `FunctionObjectValidationRuleEngineImpl`?

Reply:
@carolmariaabb no, the other implementations will fall under one of the other clauses